docs/configuration_files.md

```diff
@@ -735,21 +735,25 @@ This task is an upgrade version of the MoveToMapPokemon task. It will fetch poke
 ### Options
 [[back to top](#table-of-contents)]
 
+* `sources` - Defines whether the **WHOLE** task is enabled or not. Please bear in mind that even if the task is enabled, all or any of its sources can be disabled. (default: false)
```

Comment:
You've missed the point @YvesHenri . There is a line in configuration_files.md which reads 'sources' and it should read 'enabled'.